_translator-files/po/fr/Running-a-Server.po

```diff
@@ -386,10 +386,8 @@ msgstr "`-e ou --directoryserver`"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:140
-#, fuzzy
-#| msgid "Required for a [Registered Server](Running-a-Server#2-registered). Also required to run Jamulus as a [Directory](Directories)."
 msgid "Required for a [Registered Server](Running-a-Server#registered-mode). Also required to run Jamulus as a [Directory](Directories)."
-msgstr "Nécessaire pour un [serveur enregistré](Running-a-Server#2-registered). Également requis pour exécuter Jamulus en tant qu'[annuaire](Directories)."
+msgstr "Nécessaire pour un [serveur enregistré](Running-a-Server#1-enregistré). Également requis pour exécuter Jamulus en tant qu'[annuaire](Directories)."
```

Comment:
This link is wrong

Reply:
Unless I miss something, the link is good:
https://jamulus.io/fr/wiki/Running-a-Server#1-enregistré

Reply:
It's correct for the current, _active_ version of the website, but not for the next-release version, which is what we're translating now - see the link in English is `[Registered Server](Running-a-Server#registered-mode)` and not `[Registered Server](Running-a-Server#2-registered)` (what it used to be).

Reply:
ok, fixed I think.